ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -56,6 +57,17 @@
          task-id concept-id bulk-update-params)))))
 
 (defn handle-collection-bulk-update-event
-  "Perform update for the given concept id"
+  "Perform update for the given concept id. Log an error status of the concept
+  cannot be found."
   [context task-id concept-id bulk-update-params]
-  (data-bulk-update/update-bulk-update-task-collection-status context task-id concept-id "COMPLETE" nil))
+  (try
+    (if-let [concept (mdb2/get-latest-concept context concept-id)]
+      (data-bulk-update/update-bulk-update-task-collection-status context task-id concept-id "COMPLETE" nil)
```

Comment:
Nit: The two calls to this function having inconsistent formatting makes this a little tough to read. 
Putting the arguments all on their own line isn't pretty, but I think it makes it more obvious why you're calling this function twice.
```
(data-bulk-update/update-bulk-update-task-collection-status 
  context 
  task-id 
  concept-id 
  "COMPLETE" 
  nil)
(data-bulk-update/update-bulk-update-task-collection-status 
  context task-id 
  concept-id 
  "FAILED" 
  (format "Concept-id [%s] is not valid." concept-id))
```